var/spack/repos/builtin/packages/py-torch/package.py

```diff
@@ -432,6 +432,13 @@ class PyTorch(PythonPackage, CudaPackage, ROCmPackage):
         when="@2.0.0:2.0.1",
     )
 
+    # Use correct OpenBLAS include path under prefix
+    patch(
+        "https://patch-diff.githubusercontent.com/raw/pytorch/pytorch/pull/110063.patch?full_index=1",
+        sha256="23fb4009f7337051fc5303927ff977186a5af960245e7212895406477d8b2f66",
+        when="@:2.2",
```

Comment:
```suggestion
        when="@:2.1",
```
Upper bounds are inclusive, this will be fixed before the 2.2 release.